packages/mobile/src/invite/saga.ts

```diff
@@ -284,10 +284,13 @@ export function* redeemInviteSaga({ inviteCode }: RedeemInviteAction) {
     timeout: delay(REDEEM_INVITE_TIMEOUT),
   })
 
-  if (result === true) {
+  if (result.success === true) {
```

Comment:
I'm wondering about this. As far as I got `doRedeemInvite` returns `false | { success: boolean; newAccount: string; }`. So in case it's `false` we should see an error, no?

Reply:
That's a good call. I should do `result?.success`.